mariadb/templates/configmaps/start.sh.yaml.j2

```diff
@@ -4,34 +4,9 @@ data:
   start.sh: |+
     #!/bin/bash
     set -ex
+    trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
 
-    # Setup folders and permissions.
-    chown mysql: /var/lib/mysql
+    chown mysql: /var/lib/mysql/openstack
```

Comment:
I would rename this directory to mysql-openstack-{{{ database.cluster_name }} so we would allow for multiple mariadb deployments on the same set of nodes.